assembly/runtime.ts

```diff
@@ -1024,10 +1104,15 @@ export class ContextHelper<T extends Context> extends Context {
 }
 
 /**
- * Register a root cotnext and make it available to the runtime.
- * @param c A function that creates a new root context.
+ * Register a root context and make it available to the runtime.
```

Comment:
The lack of OOP features led me into making this change below. At this point, I was thinking on taking the root_context reference from the context objects, because that doesn't see to work as expected. Let me know what you think...

Reply:
Nevermind about this change. I reverted my changes back. 
I had originally misunderstood the OOP hack you done. There is no need for the changes I was proposing. Once needed to fix the proxy call to the onHttpCallResponse event to use the RootContextHelper wrap.